css/properties/font-width.json

```
@@ -234,6 +234,43 @@
             }
           }
         },
+        "percentage": {
```

Could we add a suffix to make it clearer that this is not a constant value?

```suggestion
        "percentage_value": {
```

---

Do you want to rename the `percentage` key on `font-stretch` too? I copied it from there.

---

Good point. It [looks like](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data+%2F%22%60%3C%5Ba-z-%5D%2B%3E%60%22%2C%2F+path%3A%2F%5Ecss%5C%2Fproperties%5C%2F%2F&type=code) we don't actually use a suffix for these cases yet, so let's keep as is.